Importers/Rhino/Speckle.Importers.Rhino/Sender.cs

```diff
@@ -19,7 +20,7 @@ public class Sender(
   ILogger<Sender> logger
 )
 {
-  public async Task<string?> Send(string projectId, string modelId, Uri serverUrl, string token)
+  public async Task<Version?> Send(string projectId, string modelId, Uri serverUrl, string token)
```

Comment:
Changed several functions to return the `Version` object they have, rather than the id. I figured it avoids primitive obsession.